### DISCUSSION TITLE
broken images in subdirectory

### DISCUSSION THREAD

**Discussion by johndhancock** [community/community#22259] [Other Feature Feedback, Questions, & Ideas]
<p>I have a “projects” subdirectory in the root of my github pages directory where individual projects live in their own subdirectories. These projects reference images that live in an images folder that is adjacent to the index.html file of each project, but images in that folder do not load on those pages. </p>
<p>For example, none of the images on this page load: <a href="https://johndhancock.github.io/projects/every-shot-dirk-ever-took/" rel="nofollow noopener">https://johndhancock.github.io/projects/every-shot-dirk-ever-took/</a></p>
<p>Folder structure is roughly: </p>
<p>-index.html</p>
<p>-projects</p>
<p>–every-shot-dirk-ever-took</p>
<p>—index.html</p>
<p>—images</p>
<p>----image.jpg</p>
<p>Any suggestions ?</p>

**Comment by johndhancock** [comment, marked as answer]
<p>Figured this out. Images started with a preceding “_”, which is not good, according to this: <a href="https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing" rel="nofollow noopener">https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing</a></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/johndhancock">@johndhancock</a>, </p>
<p>Thanks for coming back to share what work! We’re looking forward to seeing you around!</p>
